celery/worker/strategy.py

```diff
@@ -30,7 +30,7 @@ def proto1_to_proto2(message, body):
         Tuple: of ``(body, headers, already_decoded_status, utc)``
     """
     try:
-        args, kwargs = body['args'], body['kwargs']
+        args, kwargs = body.get('args', ()), body.get('kwargs', {})
```

Comment:
args are always a tuple, i don't think that returning different type in case of empty args will be correct.